src/plugins/intel_cpu/src/emitters/jit_conversion_emitters.cpp

```diff
@@ -0,0 +1,324 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "jit_conversion_emitters.hpp"
+#include "utils/bfloat16.hpp"
+#include <cpu/x64/jit_uni_eltwise.hpp>
+#include <ngraph/opsets/opset1.hpp>
+#include <nodes/eltwise.h>
+
+using namespace InferenceEngine;
+using namespace dnnl::impl::utils;
+using namespace dnnl::impl;
+using namespace dnnl::impl::cpu::x64;
+using namespace Xbyak;
+
+namespace ov {
+namespace intel_cpu {
+
+jit_convert_emitter::jit_convert_emitter(jit_generator *host, cpu_isa_t host_isa, const std::shared_ptr<ngraph::Node>& node, Precision exec_prc)
+: jit_emitter(host, host_isa, node, exec_prc) {
+    input_type = node->get_input_element_type(0);
+    output_type = node->get_output_element_type(0);
+
+    if (!mayiuse(avx512_core_bf16) && mayiuse(avx512_core))
+       emu_vcvtneps2bf16.reset(new jit_emu_vcvtneps2bf16(host, host_isa));
+}
+
+void jit_convert_emitter::validate_types() const {
+    auto is_supported_type = [this](const ov::element::Type& type) {
+        return any_of(supported_types.begin(), supported_types.end(),
+                      [&type](const ov::element::Type& supported_type) { return supported_type == type; } );
+    };
+
```

Comment:
I thought that we can only move data from src to dst vmm just in a case.. But to be honest it's really not good situation if convert has equal input and output data types. Let's add check for that. Thanks

Reply:
I have found situation we cannot throw exception
For example, if we have the following graph:
`Parameter(F32) -> TopK(I64) -> Convert(I32) - > Result(I32)`
After plugin transformation (+ precisions conversion) we get:
`Parameter(F32) -> TopK(I32) -> Convert(I32) - > Result(I32)`

Because Plugin converts I64 into I32. Also ConvertElimination couldn't remove Convert here because Convert is before Result (the transformation has this limitation).

Thus, there is Convert wit the same input and output precisions. In this case we should just move data. I reverted fix